recipe/meta.yaml

```diff
@@ -45,28 +45,25 @@ source:
     - patches/0006-fix-3.13-pickle-error-in-serialization.py-136034.patch
     # https://github.com/pytorch/pytorch/pull/137331
     - patches/0007-Allow-users-to-overwrite-ld-with-environment-variabl.patch
-    # conda-specific patch, upstream force-disables libcufile w/ TH_BINARY_BUILD
-    # for their PyPI wheel builds
-    - patches/0008-Allow-libcufile-for-conda-builds.patch
```

Comment:
For a simple removal, regenerating the patches isn't strictly necessary - it'll happen soon enough again anyway (i.e. with the next version). No change necessary, just noting that for future reference.